docs/modules/ROOT/pages/config.adoc

```diff
@@ -188,6 +188,14 @@ The optional name of the index by which the event data is to be stored. If set,
 --|string 
 |
 
+a| [[quarkus-log-handler-splunk_quarkus.log.handler.splunk.raw]]`link:#quarkus-log-handler-splunk_quarkus.log.handler.splunk.raw[quarkus.log.handler.splunk.raw]`
+
+[.description]
+--
+Determines whether the events are sent in raw mode. In case the raw event (i.e. the actual log message) is not a JSON object you need to explicitly set a source type or Splunk will reject the event (the default source type, `_json`, assumes that the incoming event can be parsed as JSON)
+--|boolean
+|`false`
```

Comment:
This file should be generated from the Javadoc of @ConfigItem by quarkus Maven plugin, so you'll need to put the doc in SplunkConfig.java so it's not lost when re-generating.

Reply:
Sorry, I didn't notice `docs` was a Maven module. Fixed (although I'm not clear if `config.adoc` needs to be updated or not).